packages/api-reference/src/components/ApiReference.vue

```diff
@@ -470,7 +472,8 @@ const changeSelectedDocument = async (
             config: mapConfiguration(config),
           },
     )
-    config.onLoaded?.(slug)
+
+    await config.onLoaded?.(slug)
```

Comment:
the await makes those calls block the further execution, right? is that really what we want here? I think they should be executed asynchronously as long was we don’t need the result of the function.